app/workers/travis_branch_monitor.rb

```diff
@@ -0,0 +1,101 @@
+require 'travis'
+
+class TravisBranchMonitor
+  include Sidekiq::Worker
+  sidekiq_options :queue => :miq_bot_glacial, :retry => false
+
+  include Sidetiq::Schedulable
+  recurrence { hourly.minute_of_hour(0, 15, 30, 45) }
+
+  include SidekiqWorkerMixin
+
+  class << self
+    private
+
+    # For this class, repos will sometimes the repo needs to be mapped to a
+    # specific gitter room, so a hash is required in some cases.
+    #
+    # This override allows for doing this in the config
+    #
+    #   travis_branch_monitor:
+    #     included_repos:
+    #       - ManageIQ/manageiq-ui-classic: ManageIQ/ui
+    #       - ManageIQ/manageiq-gems-pending: ManageIQ/core
+    #       - ManageIQ/manageiq
+    #       - ManageIQ/miq_bot
```

Comment:
Can this be simplified to

```yaml
travis_branch_monitor:
  included_repos:
    ManageIQ/manageiq-ui-classic: ManageIQ/ui
    ManageIQ/manageiq-gems-pending: ManageIQ/core
    ManageIQ/manageiq:
    ManageIQ/miq_bot:
```

This avoids the outer Array<Hash or String> and just makes it a Hash

Reply:
I think this simplifies a lof of the callers as well. For example, included_and_excluded_repos just becomes `super.keys`

Reply:
Agreed.  I think my thought process was that doing it as an array matches what was done with the other models more closely.  Not that it was a good idea to begin with.

I am all for making this simpler though, so I will muck with this and confirm that it works with the change just fine (most likely it will).

Reply:
> For example, included_and_excluded_repos just becomes `super.keys`, I think :)

Not quite, since we always return an array of `[i, e]`, but still probably will be changing it.

The other thing that usually needs to be done is "stringify" the keys, since `Config` likes returning them as symbols, regardless of the yaml provided.